### ISSUE TITLE
feat: showing best/worst coins in the coin pair card

### ISSUE THREAD

**Issue by ilbuonmarcio** [chrisleekr/binance-trading-bot#307]
I'm currently having difficulties about knowing which are the good performing coins, and the worst performing coins to change with something else.

### Describe the solution you'd like

Maybe we can have a performance ratio indicator (green good, red bad) between positive and negative trades for the single coin pair, so we can then order the coins via another filter maybe called "sort by performance asc/desc".

I'm thinking of more detailed ways to see which are good and bad coins, but this is the easiest I can find so far.

I'll try adding that next to the coin pair, maybe here, and see if that works.

![immagine](https://user-images.githubusercontent.com/21985554/132853706-1eb965b5-033c-4b4b-a9e4-4a96a757ecaa.png)

**Comment by chrisleekr** [comment]
I thought I left a message in here but I must miss it.

This is a good idea and I am happy to apply it.

> performance ratio indicator (green good, red bad) between positive and negative trades for the single coin pair

But here is a question. How do we determine it's good or bad?
So it will be like

- If the last trade was positive profit, then good?
- If the last trade was negative profit, then bad?
- If the last trade was longer than 1 day, then bad?

I think should discuss about this formula first.

**Comment by ilbuonmarcio** [comment]
Hi, thanks for your time.

We could start by having a ratio of good/bad trades, like >= 1 is good and <= 1 is bad, and then start giving more weight to latest trades and less to old ones

**Comment by chrisleekr** [comment]
Oh so like.

if the final profit of the coin is +, then good.
If the final profit of the coin is -, then bad. 

Something like this?

**Comment by ilbuonmarcio** [comment]
Yeah, that also can be it, but it can be also something else, let's think about it

**Comment by chrisleekr** [comment]
Closing this as a refactored v1 has been released. It's no longer relevant to the latest code.
